src/components/tree/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from "react";
+import { Section } from "../../types/treeType";
+
+import { info } from "./constants/info";
+import { TreeResult } from "../../types/result";
+
+type Props = {
+  section: Section;
+  show: Section | null;
+  toggleShow: (section: Section) => void;
+  treeSections: TreeResult;
+}
+
+const Button: FC<Props> = ({section, show, toggleShow, treeSections}: Props) => {
```

Comment:
Let's rename this button to `TreeMenuButton` since it only works there instead of being a generic `Button`.

Reply:
:+1: